File: Examples/PackageDiagram/Java/Core/Money.java
/** A currency amount. Part of the `Core` module. */
final class Money {
    final double amount;
    final String currency;

    Money(double amount, String currency) {
        this.amount = amount;
        this.currency = currency;
    }

    Money adding(Money other) {
        return new Money(amount + other.amount, currency);
    }
}

File: Examples/PackageDiagramDiff/Python/After/Core/money.py
class Money:
    """A currency amount. Part of the ``Core`` module."""

    def __init__(self, amount: float, currency: str):
        self.amount: float = amount
        self.currency: str = currency

    def adding(self, other: "Money") -> "Money":
        return Money(self.amount + other.amount, self.currency)
